### PR TITLE
update :shipit: 

### PR DESCRIPTION
**Pull request by shockz09**

### CODE REVIEW

**Comment by carrotfarmer** [comment]
lmao

File: author/ui-components.md [hunk 1]
```diff
@@ -130,7 +130,7 @@ class HelloWorld {
 
 Here's a list of supporetd languages:
 
-markup, css, clike, javascript, jsx, bash, batch, c, cpp, csv, dart, docker, git, groovy, ignore, java, json, kotlin, python, regex, yaml.
+markup, css, clike, javascript, jsx, bash, batch, c, cpp, csv, dart, docker, git, groovy, ignore, java, json, kotlin, python, regex, yaml,solidity.
```
Comment (gouravkhunger) [inline author/ui-components.md]: You forgot the space after `,` 😄

**Review by gouravkhunger — CHANGES_REQUESTED** [review]

**Comment by gouravkhunger** [comment]
Could you also fix the typo `supporetd` in `Here's a list of supporetd languages:` just above the language list :)

**Comment by shockz09** [comment]
> lmao

lol sry

**Review by gouravkhunger — APPROVED** [review]

**Pull request merged**